R/commonutils.R

```diff
@@ -1,3 +1,27 @@
+#' Check if a R package is installed.
+#' If installed, load the library;
+#' otherise, stop and issue warning message
+#'
+#' @param pkgNmae Package Name, a string
+#' @param level Message levels: \cr
+#' 1: message \cr
+#' 2: warn \cr
+#' 3: stop \cr
+#' @return A boolean, if the package exists.
+#' @export
+checkPackage <- function(pkgName, level) {
```

Comment:
Rename the function to better reflect what it does. I would also recommend restricting the functionality to just __check and install__ the package.